03_Predicting_Inverse_Temperature.ipynb

```diff
@@ -0,0 +1,1377 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/GDS-Education-Community-of-Practice/DSECOP/blob/daleas_module/03_Predicting_Inverse_Temperature.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment:
I think this html link also needs to be adjusted with the new name

Reply:
Thank you for your comment.   The notebook links are specific to the branch name.  I will update all of the Google Colab links simultaneously after this branch is merged with the main branch.